usaspending_api/submissions/management/commands/load_submission.py

```diff
@@ -0,0 +1,409 @@
+
+        for row in prg_act_obj_cls_data:
+            account_balances = None
+            try:
+                account_balances = AppropriationAccountBalances.objects.get(tas_rendering_label=row['tas'])
+            except:
+                continue
+
+            financial_by_prg_act_obj_cls = FinancialAccountsByProgramActivityObjectClass()
+
+            value_map = {
+                'appropriation_account_balances': account_balances,
+                'object_class': RefObjectClassCode.objects.filter(pk=row['object_class']).first(),
+                'program_activity_code': RefProgramActivity.objects.filter(pk=row['program_activity_code']).first(),
+                'create_date': timezone.now(),
+                'update_date': timezone.now()
+            }
+
+            load_data_into_model(financial_by_prg_act_obj_cls, row, value_map=value_map, save=True)
+
+        # Let's get File C information
+        db_cursor.execute('SELECT * FROM award_financial WHERE submission_id = %s', [submission_id])
+        award_financial_data = dictfetchall(db_cursor)
+        # self.logger.info('Acquired award financial data for ' + str(submission_id) + ', there are ' + str(len(award_financial_data)) + ' rows.')
+
+        for row in award_financial_data:
+            account_balances = None
+            try:
+                account_balances = AppropriationAccountBalances.objects.get(tas_rendering_label=row['tas'])
+            except:
+                continue
+
+            award_financial_data = FinancialAccountsByAwards()
+
+            value_map = {
+                'appropriation_account_balances': account_balances,
+                'object_class': RefObjectClassCode.objects.filter(pk=row['object_class']).first(),
+                'program_activity_code': RefProgramActivity.objects.filter(pk=row['program_activity_code']).first(),
+                'create_date': timezone.now(),
+                'update_date': timezone.now()
+            }
+
+            load_data_into_model(award_financial_data, row, value_map=value_map, save=True)
+
+            afd_trans = FinancialAccountsByAwardsTransactionObligations()
+
+            value_map = {
+                'financial_accounts_by_awards': award_financial_data,
+                'create_date': timezone.now(),
+                'update_date': timezone.now()
+            }
+
+            load_data_into_model(afd_trans, row, value_map=value_map, save=True)
+
+        # File D2
+        db_cursor.execute('SELECT * FROM award_financial_assistance WHERE submission_id = %s', [submission_id])
+        award_financial_assistance_data = dictfetchall(db_cursor)
+        # self.logger.info('Acquired award financial assistance data for ' + str(submission_id) + ', there are ' + str(len(award_financial_assistance_data)) + ' rows.')
+
+        for row in award_financial_assistance_data:
+            # Create LegalEntity
+            legal_entity_location_field_map = {
```

Comment:
if this dict doesn't include any variables as values, should it exist outside the for loop?